datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/GmsGraphQLEngine.java

```diff
@@ -1557,8 +1640,10 @@ private void configureCorpUserResolvers(final RuntimeWiring.Builder builder) {
     builder.type(
         "CorpUser",
         typeWiring ->
-            typeWiring.dataFetcher(
-                "relationships", new EntityRelationshipsResultResolver(graphClient)));
+            typeWiring
+                .dataFetcher("relationships", new EntityRelationshipsResultResolver(graphClient))
+                .dataFetcher(
+                    "aspects", new WeaklyTypedAspectsResolver(entityClient, entityRegistry)));
     builder.type(
```

Comment:
right below this `configureCorpUserResolvers` i believe we're missing a new configuration:
```
  private void configureResolvedAuditStampResolvers(final RuntimeWiring.Builder builder) {
    builder.type(
        "ResolvedAuditStamp",
        typeWiring ->
            typeWiring.dataFetcher(
                "actor",
                new LoadableTypeResolver<>(
                    corpUserType, (env) -> ((CorpUser) env.getSource()).getUrn())));
  }
```